test/mocha.opts

```diff
@@ -1 +0,0 @@
---require babel-register
```

Comment:
For the record, this does not need to be removed in order to address the issue from https://github.com/istanbuljs/nyc/pull/482 and https://github.com/istanbuljs/nyc/issues/509.